content/verification/va-letter-generator/release-notes/2025-04-23.md

```diff
@@ -0,0 +1,6 @@
+Two letters have been updated with the latest authorized content from VA:
+
+-Medicare Part D (Proof of Creditable Prescription Coverage) and
+-Proof of Minimum Essential Coverage
+
+The logo imagery and associated details were also updated.
```

Comment:
As a user, I might wonder what logo and details? What exactly is being updated here? I think we need to be a little bit more specific as to what logo and details are being updated here.